Strings/AppClip.jsonc

```diff
@@ -2,66 +2,66 @@
 
 	"スキャン中はカードと iPhone を動かさないようにします。カードによってスキャンが完了するまで時間がかかる場合があります。": {
 		"ja": "スキャン中はカードと iPhone を動かさないようにします。カードによってスキャンが完了するまで時間がかかる場合があります。",
-		"en": null,
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "Do not move the card or your iPhone during the scan. Depending on the card type, this process could take some time to complete.",
+		"zh-Hans": "扫瞄期间请勿移动卡片或您的 iPhone。这个过程可能需要一段时间才能完成。",
+		"zh-Hant": "掃瞄期間請勿移動卡片或您的 iPhone。此程序可能要一段時間才能完成。",
+		"zh-HK": "掃瞄期間切勿移動卡片或您的 iPhone。過程可能需要一段時間才能完成。"
 	},
 
 	"フルバージョンの App ではカードに記録されている利用履歴も確認できます。": {
 		"ja": "フルバージョンの App ではカードに記録されている利用履歴も確認できます。",
-		"en": null,
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "You can check the transaction history on the card in the full app.",
```

Comment:
I'd prefer if you used "in the full version of the app" here, as it's not descript enough or correct to just say "full app"

Reply:
"Full app" is the official wording Apple uses ([1](https://developer.apple.com/app-clips/), [2](https://developer.apple.com/design/human-interface-guidelines/app-clips/overview/), [3](https://developer.apple.com/documentation/app_clips/recommending_an_app_clip_s_corresponding_app)). I'd be happy change it, but I'm kind of a sucker for official wording from Apple haha.

Reply:
Oh, really sorry. I forgot this was about App Clips, that should work fine then. (I thought this referred to the upgraded version of the app that's purchasable in settings)

However I did just notice something else, "transaction history of the card" would be better as it's an object that's being used, rather than just referring to it as an object that stores data. (not sure if I'm explaining myself correctly here but hopefully you understand what I mean)